python/tidy/servo_tidy/tidy.py

```diff
@@ -773,6 +818,7 @@ def get_file_list(directory, only_changed_files=False, exclude_dirs=[]):
         args = ["git", "ls-files", "--others", "--exclude-standard", directory]
         file_list += subprocess.check_output(args)
         for f in file_list.splitlines():
+            f = os.path.join(*f.split("/")) if sys.platform == "win32" else f
```

Comment:
``os.path.join`` on Windows join path like ``.\\servo\\python`` with backslashes, while path in ignore list are with forward slashes, because of that ignoring paths/files doesn't work at all.